File: src/main/java/br/com/rotaractsorocabaleste/rotasales/item/ItemRepository.java
package br.com.rotaractsorocabaleste.rotasales.item;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Optional;
import java.util.Set;
import java.util.UUID;

public interface ItemRepository extends JpaRepository<Item, UUID> {

    Optional<Item> findByIdAndEventId(final UUID id, final UUID eventId);

    Set<Item> findByEventIdOrderByNameAsc(final UUID eventId);

}

File: src/main/kotlin/br/com/rotaractsorocabaleste/rotasales/dataprovider/repository/ItemRepository.kt
package br.com.rotaractsorocabaleste.rotasales.dataprovider.repository

import br.com.rotaractsorocabaleste.rotasales.core.entity.Item
import org.springframework.data.repository.CrudRepository
import java.util.*

interface ItemRepository : CrudRepository<Item, UUID> {
    fun findByEventId(eventId: UUID): List<Item>
}